src/views/PersonalSettingsPage.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+	<div class="gpoddersync_settings">
+		<SettingsSection :title="t('gpoddersync', 'Synced subscriptions')"
+			:description="t('gpoddersync', 'Podcast subscriptions that has so far been synchronized with this Nextcloud account.')">
+			<div v-if="subscriptions.length > 0">
+				<div class="sorting-container">
+					<label for="gpoddersync_sorting">Sort by:</label>
+					<Multiselect id="gpoddersync_sorting"
+						v-model="sortBy"
+						:options="sortingOptions"
+						track-by="label"
+						label="label"
+						:allow-empty="false"
+						@change="updateSorting" />
+				</div>
+				<ul>
+					<SubscriptionListItem v-for="sub in subscriptions"
+						:key="sub.url"
+						:sub="sub" />
+				</ul>
+			</div>
+			<div v-if="subscriptions.length === 0 && !isLoading">
+				<EmptyContent>
+					No subscriptions
+					<template #icon>
+						<Podcast />
+					</template>
+					<template #desc>
+						Start syncing podcasts from your favorite podcast client, such as
+						<a class="link" href="https://antennapod.org/" target="_blank">Antennapod</a>,
+						and then refresh this page to see them pop up here.
+					</template>
+				</EmptyContent>
+			</div>
+		</SettingsSection>
+	</div>
+</template>
+
+<script>
+import EmptyContent from '@nextcloud/vue/dist/Components/EmptyContent'
+import Multiselect from '@nextcloud/vue/dist/Components/Multiselect'
+import SettingsSection from '@nextcloud/vue/dist/Components/SettingsSection'
+import SubscriptionListItem from '../components/SubscriptionListItem.vue'
+
+import Podcast from 'vue-material-design-icons/Podcast'
+
+import { generateUrl } from '@nextcloud/router'
+import axios from '@nextcloud/axios'
+import { showError } from '@nextcloud/dialogs'
+
+const sortingOptions = [
+	{ label: 'Listened time (desc)', compare: (a, b) => a?.listenedSeconds < b?.listenedSeconds },
+	{ label: 'Listened time (asc)', compare: (a, b) => a?.listenedSeconds > b?.listenedSeconds },
+]
+
+export default {
+	name: 'PersonalSettingsPage',
+	components: {
+		EmptyContent,
+		Multiselect,
+		Podcast,
+		SettingsSection,
+		SubscriptionListItem,
+	},
+	data() {
+		return {
+			subscriptions: [],
+			isLoading: true,
+			sortBy: sortingOptions[0],
+			sortingOptions,
+		}
+	},
+	async mounted() {
+		try {
+			const resp = await axios.get(generateUrl('/apps/gpoddersync/personal_settings/metrics'))
+			if (!Array.isArray(resp.data.subscriptions)) {
+				throw new Error('expected subscriptions array in metrics response')
+			}
+			this.subscriptions = resp.data.subscriptions
+			this.subscriptions.sort(this.sortBy.compare)
+		} catch (e) {
+			console.error(e)
+			showError(t('gpoddersync', 'Could not fetch podcast synchronization stats'))
+		} finally {
+			this.isLoading = false
+		}
+	},
+	methods: {
+		updateSorting(sorting) {
+			this.subscriptions.sort(sorting.compare)
+		},
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+a.link {
+	text-decoration: underline;
+	color: var(--color-primary-element-light);
```

Comment:
```suggestion
	color: var(--color-primary-element);
```
For better NC25 compability, otherwise it looks like this somehow (v25.0.0rc2):
![image](https://user-images.githubusercontent.com/11487762/193455496-f61d8903-bc9e-4b06-a416-353820096042.png)
![image](https://user-images.githubusercontent.com/11487762/193455509-f22ec261-33cc-49f0-be4b-c66aa1333c0f.png)

With this change, however:
![image](https://user-images.githubusercontent.com/11487762/193455531-47d6aae8-c266-4166-b708-dc2f4af6f124.png)
![image](https://user-images.githubusercontent.com/11487762/193455543-e4edc311-1a68-4436-b0a7-2d2769b83590.png)

Reply:
Nice catch!